tests/test_http_client.rs

```diff
@@ -0,0 +1,52 @@
+#[cfg(feature = "http-client")]
+#[cfg(test)]
+mod test_http_client {
+
+    use fluvio_future::http_client::{self, ResponseExt, StatusCode};
+
+    static SERVER: &str = "https://127.0.0.1:7879";
+
+    #[async_std::test]
```

Comment:
does it need to be async_std or use default `test_async` like here: https://github.com/infinyon/future-aio/blob/master/src/net/tcp_stream.rs

Reply:
moved to test_async